playwright/containers/n8n-test-container-logs.ts

```diff
@@ -0,0 +1,374 @@
+import type { StartedTestContainer, StoppedTestContainer } from 'testcontainers';
+
+export interface LogMatch {
+	container: StartedTestContainer;
+	containerName: string;
+	message: string;
+	timestamp: Date;
+}
+
+interface WaitForLogOptions {
+	namePattern?: string | RegExp;
+	timeoutMs?: number;
+	caseSensitive?: boolean;
+}
+
+interface StreamLogMatch {
+	line: string;
+	date: Date | null;
+}
+
+/**
+ * Container helpers bound to a specific set of containers
+ */
+export class ContainerTestHelpers {
```

Comment:
The filename should probably match the class we are defining here. E.g. `n8n-test-container-helpers` instead of `-logs`